src/sidebar/services/annotations-exporter.ts

```diff
@@ -71,25 +73,23 @@ export class AnnotationsExporter {
         defaultAuthority,
       });
 
-    const annotationsText = annotations
-      .map((annotation, index) => {
-        const page = pageLabel(annotation);
-        const lines = [
-          annotation.created,
-          `Comment: ${annotation.text}`,
-          extractUsername(annotation),
-          `Quote: "${quote(annotation)}"`,
-          annotation.tags.length > 0
-            ? `Tags: ${annotation.tags.join(', ')}`
-            : undefined,
-          page ? `Page: ${page}` : undefined,
-        ].filter(Boolean);
-
-        return trimAndDedent`
-          Annotation ${index + 1}:
-          ${lines.join('\n')}`;
-      })
-      .join('\n\n');
+    const textAnnotations = annotations.map((annotation, index) => {
+      const page = pageLabel(annotation);
+      const lines = [
+        annotation.created,
+        `Comment: ${annotation.text}`,
+        extractUsername(annotation),
+        `Quote: "${quote(annotation)}"`,
+        annotation.tags.length > 0
+          ? `Tags: ${annotation.tags.join(', ')}`
+          : undefined,
+        page ? `Page: ${page}` : undefined,
+      ].filter(Boolean);
+
+      return trimAndDedent`
+        Annotation ${index + 1}:
+        ${lines.join('\n')}`;
+    });
```

Comment:
I just moved the `join` call some lines below, which made prettier change the indentation to this block